### DISCUSSION TITLE
Realtime blog feedback

### DISCUSSION THREAD

**Discussion by shorn** [supabase/supabase#4222] [Questions] [answered]
I think you might have got your axis labels backwards.

https://supabase.com/blog/2021/12/01/realtime-row-level-security-in-postgresql

First thing I thought when I saw your graph was "10 seconds to process 300 subscribers! How did they mess this up so bad?".

![image](https://user-images.githubusercontent.com/216456/144327294-3c3f1637-280a-437c-b898-c53f32133597.png)

The table makes it clear that's 350ms for 10000 users (I hope?)

**Comment by GaryAustin1** [comment, marked as answer]
I hope a funny mistake.  I reported it this morning in issues.

**Reply by w3b6x9** [reply]
here's the github issue if anyone wants to track it: https://github.com/supabase/supabase/issues/4207

**Reply by MildTomato** [reply]
apologies, it appears we drank too much coffee last night and shipped the wrong axis labels. I've submitted a PR to fix it.

**Reply by kk21** [reply]
Is "upgrade your Supabase JavaScript client to version v1.23.0" meant to be "v1.28.3"?
